simple_history/admin.py

```diff
@@ -107,7 +107,8 @@ def response_change(self, request, obj):
             return super(SimpleHistoryAdmin, self).response_change(
                 request, obj)
 
-    def history_form_view(self, request, object_id, version_id):
+    def history_form_view(self, request, object_id, version_id,
+                          extra_context=None):
```

Comment:
`extra_context` is `None` by default, so you shouldn't have to add it here. Try to add separate tests where `extra_context` has an actual value and passes through